crud.py

```diff
@@ -105,7 +105,7 @@ async def get_pay_links(wallet_ids: Union[str, List[str]]) -> List[PayLink]:
 
 
 async def update_pay_link(link_id: str, **kwargs) -> Optional[PayLink]:
-    if len(kwargs["username"]) > 0:
+    if kwargs["username"] and len(kwargs["username"]) > 0:
```

Comment:
- wouldn't `kwargs["username"]` [still throw KeyError](https://realpython.com/python-keyerror/) if `username` is not a field of `kwargs`?
- suggestion:
```py
if "username" in kwargs and len(kwargs["username"]) > 0:
```

Reply:
it does not throw, i tested it, but your still looks cleaner